CHANGELOG.md

```diff
@@ -16,6 +21,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Remodel the landing page [#170](https://github.com/IN-CORE/incore-ui/issues/170)
 - Hazard viewer previewing hazard datasets [#172](https://github.com/IN-CORE/incore-ui/pull/172)
+- Using keycloak's login page instead of custom login page [#174](https://github.com/IN-CORE/incore-ui/issues/174)
```

Comment:
do you want to remove that entry since it belongs to the Unreleased section

Reply:
Ahh shit!  Good catch.. I have removed it from the released section...